### PR TITLE
Add hotkey/toolbar icon to toggle RNNoise

### PR DESCRIPTION
**Pull request by sethbrown246**
### Checks

FEAT(ToggleRNNoise)

Hello, these are the changes I have made so far for trying to implement a global shortcut to toggle RNNoise. I believe the only thing left for me to do is to change Settings::noiseCancelMode but I was not sure how to do that. Let me know if what I have so far is good and if you have any tips on how to change Settings::noiseCancelMode to finish this issue. 

Thanks in advance! 

Implements #4189 

- [x] My commits follow the [commit guidelines](https://github.com/mumble-voip/mumble/blob/master/COMMIT_GUIDELINES.md)

### CODE REVIEW

**Comment by davidebeatrici** [comment]
The concept is correct, however I would add a shortcut to cycle through all noise cancellation methods instead.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -61,4 +61,3 @@ jobs:
       working-directory: ${{ github.workspace }}/build
       shell: bash
       run: QT_QPA_PLATFORM=offscreen ctest --output-on-failure -C $BUILD_TYPE
-
```
Comment (Krzmbrzl) [inline .github/workflows/build.yml:64]: Unrelated change

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,12 @@
+sonar.projectKey=sethbrown246_mumble
```
Comment (Krzmbrzl) [inline sonar-project.properties:1]: Unrelated changes

**Review by Krzmbrzl — CHANGES_REQUESTED** [review]
Please try to make sure that your commits follow our commit guidelines.

**Comment by Krzmbrzl** [comment]
@sethbrown246 what is your plan with this PR? Do you want to finish it?

**Comment by Krzmbrzl** [comment]
Closing this in light of #6292

**Pull request closed without merging**